### DISCUSSION TITLE
github gist 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#25] [Q&A] [answered]
IDE에서 오른쪽 클릭으로 gist로 접근이 가능해서 생성해보고 살펴보는 중인데 사용 용도가 그냥 코드 블럭을 생성하는 용도인가요?

사용하게 된다면 활용이 어떻게 가능한지 궁금합니다..!

**Comment by jongfeel** [comment, marked as answer]
코드 블럭을 생성하고 관리하는 용도가 맞습니다.
사실 유용한 코드 블록, 함수 단위 코드 관리를 통해 코멘트 주고 받고 하는 용도인데
우리나라 개발자들은 코드 블록 올려놓고 html embed된 주소 따서 블로그에 예쁘게(?) 만드는 용도로 많이 쓰는 것 같습니다.
